src/managers/TenorManager.ts

```diff
@@ -39,7 +38,7 @@ class TenorManager {
 			'key': this.apiKey,
 			'client_key': this.clientKey,
 			'contentfilter': this.contentFilter,
-			'media_filter': MEDIA_FILTER,
+			'media_filter': MediaFilter.TINYGIF,
```

Comment:
surprisingly enough it works for both minimal and basic however you need to only parse the tinygif result to reduce load on clients. The metadata associated to the obj result has enought information to generate the fullsized version

Reply:
> The metadata associated to the obj result has enought information to generate the fullsized version

it is not

Reply:
updated to use gif dimensions and not tiny gif